doc/eredis.md

```diff
@@ -12,86 +12,82 @@
 
 
 
-<a name="type-client"></a>
-### client() ###
+### <a name="type-client">client()</a> ###
 
 
 <pre><code>
 client() = pid() | atom() | {atom(), atom()} | {global, term()} | {via, atom(), term()}
-</code>
-</pre>
+</code></pre>
```

Comment:
You need to use `make edoc`. We added som HTML post-processing in the Makefile. If these linebreaks are not present, mix doesn't get it right and the hexdocs pages don't work as expected after publishing on hex.